cmd/syncthing/gui.go

```diff
@@ -210,15 +212,63 @@ func (s *apiSvc) Serve() {
 		ReadTimeout: 10 * time.Second,
 	}
 
+	s.fss = newFolderSummarySvc(s.model)
+	defer s.fss.Stop()
```

Comment:
Yeah... I think this is fine. It starts and stops together with the "parent" Serve() method on the apiSvc. I was thinking about breaking it out into something managed by itself independently, but there's the thing about it only really running when there's something listening to events from the API (for resource reasons) so it kind of makes sense to let it live in here I think.